src/test/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/initiation/InitiationJobServiceTest.java

```diff
@@ -68,7 +76,7 @@ void should_return_active_tasks_and_not_delayed_tasks() throws JSONException {
         List<CamundaTask> actualCamundaTasks = initiationJobService.getUnConfiguredTasks(SOME_SERVICE_TOKEN);
```

Comment:
Sadly, we can't test this through functional tests because we can't modify the details of the jobs created through testing and also there are competing jobs running in different env that can interfere with the tests created in your tests. I have discussed with Ravi and Alex and we agreed to validate the query.